frontend/src/pages/MessagingQueues/MQDetails/MQTables/MQTables.tsx

```diff
@@ -32,23 +32,32 @@ import {
 	MessagingQueueServicePayload,
 	MessagingQueuesPayloadProps,
 } from './getConsumerLagDetails';
+import { getTableDataForProducerLatencyOverview } from './MQTableUtils';
 
 const INITIAL_PAGE_SIZE = 10;
 
 // eslint-disable-next-line sonarjs/cognitive-complexity
 export function getColumns(
 	data: MessagingQueuesPayloadProps['payload'],
 	history: History<unknown>,
+	isProducerOverview?: boolean,
 ): RowData[] {
 	if (data?.result?.length === 0) {
 		return [];
 	}
 
+	const mergedColumns = isProducerOverview
+		? [
+				...(data?.result?.[0]?.table?.columns || []),
+				{ name: 'byte_rate', queryName: 'byte_rate' },
+		  ]
+		: data?.result?.[0]?.table?.columns;
+
```

Comment:
nit: curious if we should add this check or should we ask the BE to send the `byte_rate` column details? alternatively we can combine the column details of 1st table and 2nd table data.

i.e.
```json
{
    "data": {
        "result": [
            {
                "table": {
                    "columns": [
                        { "name": "topic", ... },
                        { "name": "service_name", ... },
                        { "name": "p99", ... },
                        { "name": "error_rate", ... },
                        { "name": "throughput", ... }
                    ]
                }
            },
            {
                "table": {
                    "columns": [
                        { "name": "service_name", ... },
                        { "name": "topic", ... },
                        { "name": "byte_rate", ... }
                    ]
                }
            }
        ]
    }
}
```

Reply:
right now, BE have some problems at their end, which is why they asked FE to do this merging of 2 tables